### PR TITLE
Declare dependencies for two Extensions.Logging packages

### PR DESCRIPTION
**Pull request by mthalman**
Port of https://github.com/dotnet/sdk/pull/35760 to the release/8.0.2xx branch.

### CODE REVIEW

**Review by NikolaMilosavljevic — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -58,11 +58,11 @@
     <MicrosoftNETHostModelVersion>8.0.0-rtm.23504.8</MicrosoftNETHostModelVersion>
     <MicrosoftExtensionsFileSystemGlobbingPackageVersion>8.0.0-rtm.23504.8</MicrosoftExtensionsFileSystemGlobbingPackageVersion>
     <MicrosoftExtensionsDependencyModelVersion>$(MicrosoftExtensionsDependencyModelPackageVersion)</MicrosoftExtensionsDependencyModelVersion>
+    <MicrosoftExtensionsLoggingVersion>$(MicrosoftExtensionsLoggingConsoleVersion)</MicrosoftExtensionsLoggingPackageVersion>
```
Comment (NikolaMilosavljevic) [inline eng/Versions.props:61]: Closing tags are different in these two properties.

File: eng/Versions.props [hunk 2]
```diff
@@ -58,11 +58,11 @@
     <MicrosoftNETHostModelVersion>8.0.0-rtm.23504.8</MicrosoftNETHostModelVersion>
     <MicrosoftExtensionsFileSystemGlobbingPackageVersion>8.0.0-rtm.23504.8</MicrosoftExtensionsFileSystemGlobbingPackageVersion>
     <MicrosoftExtensionsDependencyModelVersion>$(MicrosoftExtensionsDependencyModelPackageVersion)</MicrosoftExtensionsDependencyModelVersion>
+    <MicrosoftExtensionsLoggingVersion>$(MicrosoftExtensionsLoggingConsoleVersion)</MicrosoftExtensionsLoggingVersion>
+    <MicrosoftExtensionsLoggingAbstractionsVersion>$(MicrosoftExtensionsLoggingConsoleVersion)</MicrosoftExtensionsLoggingAbstractionsVersion>
     <MicrosoftExtensionsLoggingConsoleVersion>8.0.0-rtm.23504.8</MicrosoftExtensionsLoggingConsoleVersion>
```
Comment (NikolaMilosavljevic) [inline eng/Versions.props:63]: Would this need to be moved up, above the two new properties?

Comment (NikolaMilosavljevic) [reply eng/Versions.props:63]: Any reason to not just use the static version as in https://github.com/dotnet/sdk/pull/35760? Dependency flow would update the properties anyway.

Comment (mthalman) [reply eng/Versions.props:62]: The conflict resolution really messed things up from a simple cherry-pick.

**Review by NikolaMilosavljevic — APPROVED** [review]

**Pull request merged**